Src/BootCamp.Chapter/CityCommand.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace BootCamp.Chapter
+{
+    internal class CityCommand
+    {
+        enum Property
+        {
+            Money,
+            ItemCount
+        }
+
+        private readonly FileInfo _outputFile;
+        private readonly Property _toFilterBy;
+        private readonly Func<IEnumerable<Transaction>, string> _extrema;
+
+        public CityCommand(FileInfo outputFile, string arg1, string arg2)
```

Comment:
Consider modifying `outputFile` to a `Func<Stream>` to be invoked when it's time to write. Removes dependency on file syetem.

Reply:
It does!